doc/Language/typesystem.pod6

```diff
@@ -258,6 +258,49 @@ you must use private attribute notation instead.
     C.new.say; C.new('answer').say;
     # OUTPUT: «C.new(attr => 42)␤
     #          C.new(attr => "answer")␤»
+    
+=head4 submethod TWEAK
+
+The L<submethod|/type/Submethod> C<TWEAK> is (indirectly) called by
+L<.bless|/type/Mu#method_bless>. It
+is meant to set private and public attributes of a class after the BUILD submethod has been called. The default
+constructor L<.new|/type/Mu#method_new> defined in C<Mu> is the method that
+invokes it. Given that public accessor methods are not available in C<TWEAK>,
+you must use private attribute notation instead.
+
+The difference between BUILD and TWEAK is that not all the attributes in an object have been built.
+In addition, when there is inheritance the BUILD submethods of the child classes are called after
+the BUILD submethods of the parent classes. The TWEAK submethods are called after all the BUILD 
+submethods.
+
```

Comment:
All TWEAKs after all BUILDs or every TWEAK after its own BUILD?

Reply:
Now that is something I do not know. Be interesting to find out. Or to have my text edited to show.

Reply:
Or just change to what we know is true: every object's TWEAK happens after every object's BUILD.

Reply:
Just run a small program (see below). It's clear: each TWEAK follows directly after the BUILD, from parent to child.
```
#!/usr/bin/env perl6
class A {
    has $.a;
    submethod BUILD {
        say 'In A BUILD'
    }
    submethod TWEAK {
        say 'In A TWEAK'
    }
}
class B is A {
    has $.a;
    submethod BUILD {
        say 'In B BUILD'
    }
    submethod TWEAK {
        say 'In B TWEAK'
    }
}
class C is B {
    has $.a;
    submethod BUILD {
        say 'In C BUILD'
    }
    submethod TWEAK {
        say 'In C TWEAK'
    }
}

C.new;
# OUTPUT: 
#In A BUILD
#In A TWEAK
#In B BUILD
#In B TWEAK
#In C BUILD
#In C TWEAK
```

Reply:
Fair enough, just clarify it in the text.